File: java/006/Main.java
/*
 * ProjectEuler 006
 * Java / Main.java
 * copyright (c) 2015 Susisu
 *
 * Java SE 1.8.0_25
 * `javac Main.java -d bin`
 */

public class Main {
    public static void main(String[] args) {
        int s = 0;
        int t = 0;
        for (int n = 1; n <= 100; n++) {
            s += n;
            t += n * n;
        }
        System.out.println(s * s - t);
    }
}

File: rust/006/main.rs
/*
    ProjectEuler 006
    Rust / main.rs
    copyright (c) 2015 Susisu

    Rust 1.3.0
    `rustc main.rs -o bin/main`
*/

fn main() {
    let mut sum = 0;
    for i in 1..101 {
        for j in (i + 1)..101 {
            sum += 2 * i * j;
        }
    }
    println!("{}", sum);
}
